network/network/src/main/java/bisq/network/p2p/common/HandlerLifecycle.java

```diff
@@ -17,7 +17,7 @@
 
 package bisq.network.p2p.common;
 
-public interface LifecycleHandler {
+public interface HandlerLifecycle {
```

Comment:
Maybe we should move that to common module next to the Service interface. 
I don't have a good name suggestion. In mobile we use `LifecycleAware` but not super happy with that but did not find a better one. If you have some suggestion, please share. 

We could also rename Service to AsyncService and use Service for that one... If we do that lets do it in a dedicated PR with only that refactoring.